comps/dataprep/src/utils.py

```diff
@@ -233,20 +234,28 @@ async def load_docx(docx_path):
 async def load_ppt(ppt_path):
     """Load ppt file."""
     print("Converting ppt file to pptx file...")
-    pptx_path = ppt_path + "x"
+
+    temp_dir = tempfile.gettempdir()
+    base_name = os.path.splitext(os.path.basename(ppt_path))[0]
+    pptx_path = os.path.join(temp_dir, base_name + ".pptx")
+
     subprocess.run(
```

Comment:
Running Libreoffice is surely slow => why this is not using async APIs, so that something else can be done in the meanwhile?